sdk/src/main/scala/com/horizen/api/http/SidechainNodeApiRoute.scala

```diff
@@ -90,6 +90,25 @@ case class SidechainNodeApiRoute(peerManager: ActorRef,
     }
   }
 
+  def disconnect: Route = (post & path("disconnect")) {
+    entity(as[ReqDisconnect]) {
+      body =>
+        var address = body.host + ":" + body.port
+        val maybeAddress = addressAndPortRegexp.findFirstMatchIn(address)
+        maybeAddress match {
+          case None =>
+            ApiResponseUtil.toResponse(ErrorInvalidHost("Incorrect host and/or port.", None))
+          case Some(addressAndPort) =>
+            val host = InetAddress.getByName(addressAndPort.group(1))
+            val port = addressAndPort.group(2).toInt
+            val peerAddress = new InetSocketAddress(host, port)
+            peerManager ! RemovePeer(peerAddress)
+            networkController ! Blacklisted(peerAddress)
```

Comment:
double check if there is the necessity to introduce an additional method to just disconnect (BlackListed seems to not really ban)